importer/SeBbrSv.py

```diff
@@ -183,7 +184,12 @@ def set_inception(self):
         if self.has_non_empty_attribute("byggar"):
             year_parsed = utils.parse_year(self.byggar)
             if year_parsed is not None:
```

Comment:
no need for "is not None". Just use `if year_parsed` (since "0" is not a valid value anyway it shouldn't cause an issue)